app/helpers/invoices_helper.rb

```diff
@@ -0,0 +1,25 @@
+module InvoicesHelper
+  def get_items
+    Item.all.map{ |item|[item.name,item.id]}
+  end
+
+  def get_clients
+    current_user.company.clients.map{ |client|[client.name,client.id]}
+  end
+
+  def date_for_display(date)
```

Comment:
Move this method to `ApplicationHelper`. Rename it to `format_date`

It should accept another parameter `format` with default value `%m/%d/%Y` and use it.